mining-network-prop/ANSWERS.md

```diff
@@ -7,11 +7,11 @@ A: **Why**: "the nonce value in the header field is only 32 bit, which is a triv
 Q: Greg Maxwell mentioned that miners are hesitant to connect with one another directly, even though it would speed up block propagation. Why?  
 A: If your competitor knows your IP address they are more easily able to DoS you.
 
-Q: How is a high block-orphaning rate the best indicator that block propagation times are too high?  
+Q: How is a high stale block rate the best indicator that block propagation times are too high?  
 A: If a miner isn’t able to quickly discover new blocks, they are more likely to find a valid block on an out-of-date chain, and end up publishing a valid block that is behind the chain tip.
 
-Q: Alternatively, while a low block-orphaning rate can be an indicator that block propagation times are good, it could also be an indicator of something more problematic. What could be a bad reason for there to be very few orphans?  
-A: It could be the case that there's very little orphaning because mining is so centralized that there’s no competition to get orphaned
+Q: Alternatively, while a low stale block rate can be an indicator that block propagation times are good, it could also be an indicator of something more problematic. What could be a bad reason for there to be very few stale blocks?  
+A: It could be the case that there are very few stale blocks because mining is so centralized that there’s no competition blocks to become stale.
```

Comment:
```suggestion
A: It could be the case that there are very few stale blocks because mining is so centralized that there’s not enough competition for blocks to become stale.
```